pom.xml

```diff
@@ -33,6 +34,40 @@
 		</dependency>
 	</dependencies>
 
+	<build>
+		<plugins>
+			<!-- Maven Assembly Plugin -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<version>${assembly.version}</version>
+				<configuration>
+					<!-- get all project dependencies -->
+					<descriptorRefs>
+						<descriptorRef>jar-with-dependencies</descriptorRef>
+					</descriptorRefs>
+					<!-- MainClass in mainfest make a executable jar -->
+					<archive>
+						<manifest>
+							<mainClass>com.twimba.fluentd4log4j.FluentdAppender</mainClass>
```

Comment:
This makes no sense at all... `com.twimba.fluentd4log4j.FluentdAppender` doesn't even contain a main class! :/